src/base/datetime/calendar/header/index.stories.js

```diff
@@ -0,0 +1,41 @@
+import { BaseDateTimeCalendarHeader } from "./index.ts";
+import { storiesOf } from "@storybook/web-components";
+
+storiesOf("base/datetime/calendar/header", module)
+  .add("Base EN", () => {
+    const root = document.createElement("div");
+
+    const calendarHeaderEN = new BaseDateTimeCalendarHeader({ language: "en" });
```

Comment:
@dong0 
Because don't supporting set prop on constructor.
Please change to using document.createElement() for creating an DOM.